src/blob_gc_job.cc

```diff
@@ -508,6 +520,32 @@ Status BlobGCJob::RewriteValidKeyToLSM() {
     s = Status::OK();
   }
 
+  mutex_->Lock();
+  auto cf_id = blob_gc_->column_family_handle()->GetID();
+  for (auto blob_file : dropped) {
+    auto blob_storage = blob_file_set_->GetBlobStorage(cf_id).lock();
+    if (blob_storage) {
+      auto file = blob_storage->FindFile(blob_file.first).lock();
+      if (!file) {
+        ROCKS_LOG_ERROR(db_options_.info_log,
+                        "Blob File %" PRIu64 " not found when GC.",
+                        blob_file.first);
+        continue;
+      }
+      SubStats(stats_, cf_id, file->GetDiscardableRatioLevel(), 1);
+      file->UpdateLiveDataSize(-blob_file.second);
+      fprintf(stderr, "update file %lu dropped %lu %lf", blob_file.first,
```

Comment:
doubt why not use `ROCKS_LOG_BUFFER` macro here?

Reply:
just forget to clean the log used for debug